flexbox/src/main/java/com/google/android/flexbox/FlexboxLayout.java

```diff
@@ -614,64 +623,91 @@ private void determineMainSize(@FlexDirection int flexDirection, int widthMeasur
      */
     private int expandFlexItems(FlexLine flexLine, @FlexDirection int flexDirection,
             int maxMainSize, int paddingAlongMainAxis, int startIndex) {
+        int childIndex = startIndex;
```

Comment:
expandFlexItems doesn't need to be changed unless layout_maxHeight/layout_maxWidth are implemented.
Could you revert the change for the expandFlexItems?